### PR TITLE
fix(rename): id alias frontend

### PR DESCRIPTION
**Pull request by safeamiiir**
Frontend renaming related to this PR: https://github.com/AqueductHub/aqueductcore/pull/143

### CODE REVIEW

**Review by str-anger — CHANGES_REQUESTED** [review]
Probably may become better with some minor changes.

File: aqueductcore/frontend/src/__mocks__/queries/experiment/getExperimentFilesById.tsx [hunk 1]
```diff
@@ -1,22 +1,22 @@
 import { GET_EXPERIMENT_FILES_BY_ID } from "API/graphql/queries/experiment/getExperimentFilesById";
-import { ExperimentsDataMock, alias } from "__mocks__/ExperimentsDataMock";
+import { ExperimentsDataMock, eid } from "__mocks__/ExperimentsDataMock";
```
[the remark this answers is not in the document]
Comment (safeamiiir) [reply aqueductcore/frontend/src/__mocks__/queries/experiment/getExperimentFilesById.tsx]: No it's a sample for `eid` I've changed it into `sample_eid`

**Comment by safeamiiir** [comment]
Thanks @str-anger for all better naming! 😁

**Review by str-anger — APPROVED** [review]
Good to go!

**Pull request merged**